integration/README.md

```diff
@@ -0,0 +1,37 @@
+# Requirements
+
+```
+go get -u github.com/giantswarm/e2e-harness
+```
+
+## environment variables example
+```
+export CLUSTER_NAME="test-e2e"
+export COMMON_DOMAIN_GUEST="k8s.godsmack.westeurope.azure.gigantic.io"
```

Comment:
Why we need that, when we can do `k8s.$COMMON_DOMAIN_GUEST_NO_K8S`?

Reply:
we could even have
```
export AZURE_LOCATION="westeurope"
export COMMON_DOMAIN_RESOURCE_GROUP="godsmack"
export COMMON_DOMAIN_GUEST_NO_K8S="${COMMON_DOMAIN_RESOURCE_GROUP}.${AZURE_LOCATION}.azure.gigantic.io"
export COMMON_DOMAIN_GUEST="k8s.${COMMON_DOMAIN_GUEST_NO_K8S}"
```

Reply:
Yes, for me that's better